pkg/networkengine/util/ipset/ipset.go

```diff
@@ -20,120 +20,103 @@
 package ipsetutil
 
 import (
-	"sync"
+	"fmt"
 
-	"github.com/gonetx/ipset"
+	"github.com/vishvananda/netlink"
 	"k8s.io/klog/v2"
 )
 
 type IPSetInterface interface {
-	List(options ...ipset.Option) (*ipset.Info, error)
+	List() (*netlink.IPSetResult, error)
 	Name() string
-	Add(entry string, options ...ipset.Option) error
-	Del(entry string, options ...ipset.Option) error
+	Add(entry *netlink.IPSetEntry) error
+	Del(entry *netlink.IPSetEntry) error
 	Flush() error
 	Destroy() error
 }
 
 type ipSetWrapper struct {
-	ipset.IPSet
-}
-
-var (
-	once sync.Once
-)
-
-func check() error {
-	if err := ipset.Check(); err != nil {
-		klog.ErrorS(err, "error on ipset.Check")
-		return err
-	}
-	if klog.V(5).Enabled() {
-		klog.V(5).InfoS("ipset.Check succeeded")
-	}
-	return nil
+	setName string
 }
 
 func New(setName string) (IPSetInterface, error) {
-	var err error
-	once.Do(func() {
-		err = check()
+	err := netlink.IpsetCreate(setName, "hash:net", netlink.IpsetCreateOptions{
+		Replace: true,
```

Comment:
how big will `MaxElements` size be used by default?

Reply:
![image](https://user-images.githubusercontent.com/15940738/233882060-08f101d0-bb75-4f7b-b915-6afb37a37c6e.png)